plugins/woocommerce/tests/e2e-pw/tests/smoke-tests/upload-plugin.spec.js

```diff
@@ -1,91 +1,56 @@
 const {
 	ADMINSTATE,
-	ADMIN_USER,
-	ADMIN_PASSWORD,
 	GITHUB_TOKEN,
 	PLUGIN_NAME,
 	PLUGIN_REPOSITORY,
 } = process.env;
 const { test, expect } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
 const path = require( 'path' );
 const {
-	createPlugin,
 	deletePlugin,
-	downloadZip,
 	deleteZip,
-	getLatestReleaseZipUrl,
+	downloadZip,
+	installPluginThruWpCli,
 } = require( '../../utils/plugin-utils' );
 
-const adminUsername = ADMIN_USER ?? 'admin';
-const adminPassword = ADMIN_PASSWORD ?? 'password';
-
-let pluginZipPath;
+let pluginPath;
 let pluginSlug;
 
 test.describe( `${ PLUGIN_NAME } plugin can be uploaded and activated`, () => {
-	// Skip test if PLUGIN_REPOSITORY is falsy.
 	test.skip(
 		! PLUGIN_REPOSITORY,
-		`Skipping this test because value of PLUGIN_REPOSITORY was falsy: ${ PLUGIN_REPOSITORY }`
+		`SKIPPED: PLUGIN_REPOSITORY is falsy: ${ PLUGIN_REPOSITORY }`
```

Comment:
This makes the test skippable especially on PR smoke tests, and other cases where we don't necessarily need to upload any other plugin.